src/main/java/uk/gov/ons/census/casesvc/model/entity/Event.java

```diff
@@ -28,4 +33,13 @@
   @Column
   @Enumerated(EnumType.STRING)
   private EventType eventType;
+
+  @Column private UUID caseId;
```

Comment:
This column shouldn't be here because we can always infer the caseId from the Event -> UICQIDLink -> Case relationship. In the case of unaddressed, we might end up missing some of the events associated with a case.

Reply:
done